doc/man1/openssl-smime.pod.in

```diff
@@ -177,29 +177,31 @@ If not specified triple DES is used. Only used with B<-encrypt>.
 
 =item B<-nointern>
 
-When verifying a message normally certificates (if any) included in
-the message are searched for the signing certificate. With this option
-only the certificates specified in the B<-certfile> option are used.
-The supplied certificates can still be used as untrusted CAs however.
+When verifying a message, normally certificates (if any) included in
+the message are searched for the signing certificate. With this option,
+only the certificates given with the B<-certfile> option are used.
+However, unless the B<-nochain> option is given, the certificates included in
+the message are still used for chain building as certificates of untrusted CAs.
 
 =item B<-noverify>
 
 Do not verify the signers certificate of a signed message.
 
 =item B<-nochain>
 
-Do not do chain verification of signers certificates; that is, do not
-use the certificates in the signed message as untrusted CAs.
+During chain verification of signer certificates, do not use the certificates
+in the signed message as untrusted CA certificates for chain building.
```

Comment:
Adapted as follows, in order to make more clear why CA certs are involved here and to avoid the term 'issuer CA', which may be understood as only those CAs that issue EE certs:
```
When verifying the signer certificates of a signed message, do not use the
certificates in the message as candidate CA certificates for chain building.
```